CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ This file is generated automatically. Add individual changelog entries to the 'c
 
 This changelog documents all notable changes to VAST and is updated on every release. Changes made since the last release are in the [changelog/unreleased directory][unreleased].
 
+## [v1.1.2]
+
+### :beetle: Bug Fixes
+
+- Fixed a race condition that would cause queries to become stuck when an importer would time out during the meta index lookup.
```

Comment:
Did you mean to say "exporter" by any chance?

Reply:
Indeed, good catch

Reply:
The same problem is also in the specific file in the changelog subdir.